template.tpl

```diff
@@ -1207,18 +1207,24 @@ ___TEMPLATE_PARAMETERS___
     "groupStyle": "ZIPPY_CLOSED",
     "subParams": [
       {
-        "type": "SELECT",
+        "type": "SIMPLE_TABLE",
```

Comment:
I guess in classic software versioning, this would be a breaking change. I'm not really sure how that applies to GTM Tags, since you get a UI telling you everything that'll break, so maybe we don't need to worry.
I think this is a better change, but just curious about the user experience. I guess we could make it v2?

Reply:
It is indeed a breaking change.. my confusion was that users can still provide the "same input", in the sense that they can point to same GTM variable, just through a row of the table. However, as you said, technically it is a breaking change and it probably requires users to do this manually to their affected tags when updating. Let's go for v2!